### PR TITLE
Fixed an issue where npm crashed make setup

### PR DESCRIPTION
**Pull request by engelhamer**
## Description
Had not been changed in 2 years:: npm now comes with nodejs, attempting to install it in reverse order causes unmet or broken dependency errors.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -94,14 +94,14 @@ setup-no-input:
...
-	sudo apt install npm -y
+	sudo apt install nodejs -y
 	sudo npm install npm@latest -g
 	sudo apt install nodejs python3 python3-pip libpq-dev python3-dev postgresql postgresql-contrib rabbitmq-server python3-setuptools sshpass -y
 
 	make setup-venv requirements_file=local.txt
 
 	# Istall nodejs dependencies.
-	npm install --prefix ./src/vue
+	npm install --legacy-peer-deps --prefix ./src/vue
```
Comment (engelhamer) [inline Makefile:104]: This is likely required because the package lock was created using an outdated npm version.

File: src/vue/package.json [hunk 2]
```diff
@@ -13,28 +13,28 @@
...
-    "core-js": "^3.6.4",
-    "vue": "^2.6.11",
-    "vue-router": "^3.1.6",
-    "vuex": "^3.1.3",
     "@sentry/browser": "^5.14.2",
     "@sentry/integrations": "^5.14.2",
     "axios": "^0.18.1",
     "bootstrap": "^4.4.1",
     "bootstrap-vue": "^2.16.0",
+    "core-js": "^3.6.4",
```
Comment (mjvkeulen) [inline src/vue/package.json:21]: [NIT]: There no changes in this file right? Prob better to not commit

File: Makefile (hunk 1 above)
Comment (mjvkeulen) [reply Makefile:104]: I don't quite follow, we stopped using the package-lock right?

File: src/vue/package.json (hunk 2 above)
Comment (engelhamer) [reply src/vue/package.json:21]: It is now sorted alphabetically

File: Makefile (hunk 1 above)
Comment (engelhamer) [reply Makefile:104]: Sorry, I mean the `package.json`

**Review by engelhamer — COMMENTED** [review]
.

**Review by Larspolo — APPROVED** [review]

**Review by mjvkeulen — APPROVED** [review]

**Pull request merged**